lxd/sys/apparmor.go

```diff
@@ -25,4 +27,36 @@ func (s *OS) initAppArmor() {
 
 	/* Detect AppArmor stacking support */
 	s.AppArmorStacking = util.AppArmorCanStack()
```

Comment:
So since we're using a local, private function for haveMacAdmin here, can we do the same for aaCanStack?
We're not using util.AppArmorCanStack anywhere other than from this file, so I'm not sure it makes sense to have it be a separate exported function under util.

Reply:
Sure, I did that in b8927b305775f8148903eb234f41303136bd2e17.